src/generator/ruleDetailsSkeleton

```diff
@@ -0,0 +1,19 @@
+# _RULENAME_
+
+Checks if...
+
+## ❓ Why it's good to follow this rule?
+
+Content here...
+
+## 😱 Examples of code for which this rule will throw a warning
+
+Example of bad code...
+
```

Comment:
I would add a code block here with a todo comment

Reply:
Added in last commit